view/utils/tablesorter-time-range-comarator.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+function contains(a, searchString) {
```

Comment:
Also tests would be welcome... for example:

[ "6 h 35 m", "6 h 34 m", "34 m", "14 h 41 m", "20 h", "< 1m", "10 m", "1 h"]

should sort in the following way:

["< 1m", "10 m", "34 m", "1 h", "6 h 35 m", "6 h 34 m", "14 h 41 m", "20 h"]